geode-core/src/main/java/org/apache/geode/distributed/internal/membership/gms/messenger/JGroupsMessenger.java

```diff
@@ -845,7 +862,7 @@ public void waitForMessageState(ID sender, Map<String, Long> state)
    * @return the new message
    */
   org.jgroups.Message createJGMessage(Message<ID> gfmsg, JGAddress src, ID dst,
-      short version) {
+      short version) throws IOException {
```

Comment:
Why would a method that does no I/O, throw an `IOException`? Seems like this should be throwing some other kind of exception.